pycds/alembic/extensions/operation_plugins.py

```diff
@@ -108,6 +114,60 @@ def drop_constraint_if_exists(operations, operation):
     )
 
 
+@Operations.register_operation("create_index_if_not_exists")
+@BatchOperations.register_operation(
+    "create_index_if_not_exists", "batch_create_index_if_not_exists"
+)
+class CreateIndexIfNotExists(CreateIndexOp):
+    """
+    Represent a create index if not exists operation.
+
+    This operation's class is identical to DropConstraintOp except for
+    class method names which become operation names. The new names dispatch
+    to the old names.
+    """
```

Comment:
I don't understand the role of the "old names" or "legacy names" in this operation. That may just be my lack of context.